dmap_adv_markers/3/dmap_adv_markers/userconfig/DMapAdvMarkers/config.txt
//
//  markerslist <marker name> [<marker name> ...]
//  colorslist <color name> [<color name> ...]
//  marker <marker name> <key code> [shift] [ctrl] [alt]
//  color <color name> <key code> [shift] [ctrl] [alt]
//  send <channel name> <key code> [shift] [ctrl] [alt]
//
//
//  <marker name> - any valid marker (from CfgMarkers)
//  <color name> - any valid marker color name (from CfgMarkerColors)
//
//  Color name Default means to use the color, specified by a marker
//
//
//  Possible channel names: 
//    global
//    side
//    command
//    group
//    vehicle
//    direct
//

// markers list, switchable by cursor keys
markerslist mil_dot hd_dot hd_unknown hd_warning hd_join hd_pickup hd_end hd_start hd_destroy hd_ambush hd_arrow hd_flag hd_objective o_unknown o_armor o_inf
//colorslist Default ColorBlack ColorGreen

// send to channel hotkeys
send side 40 ctrl              // ctrl+'
send group 39 ctrl             // ctrl+;
send vehicle 38 ctrl           // ctrl+L



// marker color hotkeys
color ColorBlack 2 ctrl        // ctrl+1
color ColorRed 3 ctrl          // ctrl+2
color ColorGreen 4 ctrl        // ctrl+3
color ColorBlue 5 ctrl         // ctrl+4
color ColorYellow 6 ctrl       // ctrl+5
color ColorOrange 7 ctrl       // ctrl+6
color ColorWhite 8 ctrl        // ctrl+7
color ColorPink 9 ctrl         // ctrl+8
color ColorBrown 10 ctrl       // ctrl+9
color Default 11 ctrl       // ctrl+0



// marker type hotkeys
marker mil_dot 16 ctrl          // ctrl+Q
marker hd_dot 17 ctrl        // ctrl+W
marker hd_unknown 18 ctrl      // ctrl+E
marker hd_warning 19 ctrl      // ctrl+R
marker hd_join 20 ctrl    // ctrl+T
marker hd_pickup 30 ctrl         // ctrl+A
marker hd_end 31 ctrl        // ctrl+S
marker hd_start 32 ctrl       // ctrl+D
marker hd_destroy 33 ctrl      // ctrl+F
marker hd_ambush 34 ctrl        // ctrl+G
marker hd_flag 44 ctrl          // ctrl+Z
marker hd_objective 50 ctrl       // ctrl+M
marker o_unknown 49 ctrl         // ctrl+N
marker o_armor 35 ctrl      // ctrl+H
marker o_inf 48 ctrl      // ctrl+B
// marker hd_dot 41 ctrl          // ctrl+~






//  /* KEY LIST // Tastendruckzuweisung // Le key de la baguette // Обозначения клавиш // Zorro key // Lista klawiszy // Billentyû lista // whatever */
//  /* ================================================================================================================= */
//  /*
//  		KEY LIST FOR US KEYBOARD LAYOUT BELOW THIS LIST
//  
//  
//  shift
//  ctrl
//  alt
//  
//  ESC = 1 
//  F1 = 59 
//  F2 = 60 
//  F3 = 61 
//  F4 = 62 
//  F5 = 63 
//  F6 = 64 
//  F7 = 65 
//  F8 = 66 
//  F9 = 67 
//  F10 = 68 
//  F11 = 87 
//  F12 = 88 
//  PRINT = 183 
//  SCROLL = 70 
//  PAUSE = 197 
//  ^ = 41 
//  1 = 2 
//  2 = 3 
//  3 = 4 
//  4 = 5 
//  5 = 6 
//  6 = 7 
//  7 = 8 
//  8 = 9 
//  9 = 10 
//  0 = 11 
//  ß = 12 
//  ´ = 13 
//  Ü = 26 
//  Ö = 39 
//  Ä = 40 
//  # = 43 
//  < = 86 
//  , = 51 
//  . = 52 
//  - = 53 
//  + = NOT DEFINED
//  POS1 = 199 
//  TAB = 15 
//  ENTER = 28 
//  DELETE = 211 
//  BACKSPACE = 14 
//  INSERT = 210 
//  END = 207 
//  PAGEUP = 201 
//  PAGEDOWN = 209 
//  CAPS = 58 
//  A = 30 
//  B = 48 
//  C = 46 
//  D = 32 
//  E = 18 
//  F = 33 
//  G = 34 
//  H = 35 
//  I = 23 
//  J = 36 
//  K = 37 
//  L = 38 
//  M = 50 
//  N = 49 
//  O = 24 
//  P = 25 
//  Q = 16 
//  U = 22 
//  R = 19 
//  S = 31 
//  T = 20 
//  V = 47 
//  W = 17 
//  X = 45 
//  Y = 44 
//  Z = 21 
//  SHIFTL = 42 
//  SHIFTR = 54 
//  UP = 200 
//  DOWN = 208 
//  LEFT = 203 
//  RIGHT = 205 
//  NUM_0 = 82 
//  NUM_1 = 79 
//  NUM_2 = 80 
//  NUM_3 = 81 
//  NUM_4 = 75 
//  NUM_5 = 76 
//  NUM_6 = 77 
//  NUM_7 = 71 
//  NUM_8 = 72 
//  NUM_9 = 73 
//  NUM_+ = 78 
//  NUM = 69 
//  NUM_/ = 181 
//  NUM_* = 55 
//  NUM_- = 74 
//  NUM_, = 83 
//  NUM_ENTER = 156 
//  STRGL = 29 
//  STRGR = 157  
//  WINL = 220 
//  WINR = 219  
//  ALT = 56 
//  SPACE = 57 
//  ALTGR = 184 
//  APP = 221 
//  */
//  
//  
//  /* KEY LIST // English (US) 104-key    provided by tcp */ 
//  /* ================================================================================================================= */
//  /*
//  ESC = 1 
//  F1 = 59 
//  F2 = 60 
//  F3 = 61 
//  F4 = 62 
//  F5 = 63 
//  F6 = 64 
//  F7 = 65 
//  F8 = 66 
//  F9 = 67 
//  F10 = 68 
//  F11 = 87 
//  F12 = 88 
//  PRINT = 183 
//  SCROLL = 70 
//  PAUSE = 197 
//  ` = 41  
//  1 = 2 
//  2 = 3 
//  3 = 4 
//  4 = 5 
//  5 = 6 
//  6 = 7 
//  7 = 8 
//  8 = 9 
//  9 = 10 
//  0 = 11 
//  - = 12 
//  = = 13 
//  , = 51 
//  . = 52 
//  / = 53
//  ; = 39
//  ' = 40
//  [ = 26
//  ] = NOT DEFINED
//  \ = 43
//  HOME = 199 
//  TAB = 15 
//  ENTER = 28 
//  DELETE = 211 
//  BACKSPACE = 14 
//  INSERT = 210 
//  END = 207 
//  PAGEUP = 201 
//  PAGEDOWN = 209 
//  CAPS = 58 
//  A = 30 
//  B = 48 
//  C = 46 
//  D = 32 
//  E = 18 
//  F = 33 
//  G = 34 
//  H = 35 
//  I = 23 
//  J = 36 
//  K = 37 
//  L = 38 
//  M = 50 
//  N = 49 
//  O = 24 
//  P = 25 
//  Q = 16 
//  R = 19 
//  S = 31 
//  T = 20 
//  U = 22 
//  V = 47 
//  W = 17 
//  X = 45 
//  Y = 21 
//  Z = 44 
//  SHIFTL = 42 
//  SHIFTR = 54 
//  UP = 200 
//  DOWN = 208 
//  LEFT = 203 
//  RIGHT = 205 
//  NUM_0 = 82 
//  NUM_1 = 79 
//  NUM_2 = 80 
//  NUM_3 = 81 
//  NUM_4 = 75 
//  NUM_5 = 76 
//  NUM_6 = 77 
//  NUM_7 = 71 
//  NUM_8 = 72 
//  NUM_9 = 73 
//  NUM_+ = 78 
//  NUM = 69 
//  NUM_/ = 181 
//  NUM_* = 55 
//  NUM_- = 74 
//  NUM_. = 83
//  NUM_ENTER = 156 
//  CTRLL = 29 
//  CTRLR = 157 
//  WINL = 219 
//  WINR = 220 
//  ALTL = 56
//  ALTR = 184
//  SPACE = 57 
//  APP = 221
//  */


dmap_adv_markers/1/dmap_adv_markers/settings.hpp
#ifndef __SETTINGS_HPP_
#define __SETTINGS_HPP_

#define USERCONFIG_PATH "\userconfig\DMapAdvMarkers\config.txt"

#define DISTANCE_TO_MARKER_TO_DELETE_SQR 0.0008

#define CLIENT_MARKER_COLOR_AR ["Default", "ColorBlack", "ColorRed", "ColorGreen", "ColorBlue", "ColorYellow", "ColorOrange", "ColorWhite", "ColorPink", "ColorBrown", "ColorKhaki"]
#define CLIENT_MARKER_THICKNESS_AR [0.5, 1, 2, 3, 5, 8, 12, 18, 27, 40]
//, 60]

//#define CLIENT_MARKER_DEFAULT_COLOR 0
#define CLIENT_MARKER_DEFAULT_THICKNESS 4



#define LINE_MARKER_CHANNEL_TO_TYPE_AR ["mil_circle", "mil_dot", "mil_arrow2", "mil_triangle", "mil_box", "mil_arrow"];

#define MARKER_CHANNEL_TO_CHAR_AR ["G", "S", "C", "g", "V", "D"];



#define MTAG_DEFAULT_OPTIONS [MTAG_ALWAYS_ON, MTAG_OFF, MTAG_ALWAYS_ON, MTAG_ALWAYS_OFF]

#define TEMPORARY_MARKERS_TIMEOUT 5

#endif
